Modules/Logger/src/selxLogger.cxx

```diff
@@ -24,12 +24,14 @@ namespace selx {
 
 Logger
 ::Logger() {
-  this->m_LoggerImpl = LoggerImplPointer( new LoggerImpl() );
+  this->m_LoggerImpl = std::make_unique<LoggerImpl>();
```

Comment:
Is this just style or does it have functional implications? If so, I'd be interested in hearing which?

Reply:
@coertmetz Thanks for your suggestion to use std::make_unique. I agree that in principle that would be preferable to a 'raw' new. However, at the moment SuperElastix still needs to support C++11 (as I just checked with @FBerendsen), and std::make_unique was only introduced with C++14.